src/index.ts

```diff
@@ -381,8 +386,23 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
   validation.validateNotBlank(config.apiKey, 'API key required');
   let initializationError: Error | undefined;
   const instance = EppoJSClient.instance;
-  const { apiKey, persistentStore, baseUrl, maxCacheAgeSeconds, updateOnFetch } = config;
+  const { apiKey, persistentStore, baseUrl, maxCacheAgeSeconds, updateOnFetch, forceReinitialize } =
+    config;
   try {
+    if (EppoJSClient.initialized) {
+      if (forceReinitialize) {
+        applicationLogger.warn(
+          'Eppo SDK is already initialized. Since forceReinitialize is true, reinitializing.',
```

Comment:
i'd change this to an `info` instead of a `warn`. also minor rewording: `Eppo SDK is already initialized. Reinitializing since forceReinitialize is true.`

Reply:
TBH none of these logs are getting emitted anyways since this is the browser 😄

Reply:
hmm would be nice to use a tool like `debug` so consumers of the library can choose whether or not to emit console logging